src/collectors/uniswapx_order_collector.rs

```diff
@@ -44,21 +45,15 @@ impl FromStr for OrderType {
     }
 }
 
-impl ToString for OrderType {
-    fn to_string(&self) -> String {
+impl fmt::Display for OrderType {
```

Comment:
impl Display gives you ToString for free

Reply:
nit: resolve Display import at top of file?